src/components/VmActions/index.js

```diff
@@ -132,26 +132,12 @@ class VmActions extends React.Component {
     const status = vm.get('status')
     const onStart = (isPool ? onStartPool : onStartVm)
 
-    // TODO: On the card list page, the VM's consoles would not have been fetched yet,
-    // TODO: so the tooltip on the console button will be blank.
-    let consoleProtocol = ''
-    if (!vm.get('consoles').isEmpty()) {
-      const vConsole = vm.get('consoles').find(c => c.get('protocol') === 'spice') || vm.getIn(['consoles', 0])
-      const protocol = vConsole.get('protocol').toUpperCase()
-      consoleProtocol = msg.openProtocolConsole({ protocol })
-    }
-
-    if (vm.get('consoleInUse')) {
```

Comment:
"console in use" functionality was not marked for removal

Reply:
Hi @rszwajko 
The quoted text from the tooltips document.

>  Similar comment goes for these infotips, I would get rid of them unless they are there for accessibility.

since `consoleProtocol` variable used only for infotip which removed, then I removed all the lines that deals with that variable, include "console in use.
the console infotip is not mentioned explicitly but IMHO if I wouldn't remove that inotip it would create inconsistency.
WDYT?

Reply:
OK. I took a second look at this code and it seems that we have a mix of features and bugs here. 
Features:
1. when console is closed the tool tip is `Open {protocol} console`  (where protocol is SPICE or first available)
2. when console is open the tool tip is `Console in use`

Bugs:
1. it doesn't work in 'on card' scenario - the tool tip is set on main action whereas `VmDropdownActions` uses `action.items`. This seems the only scenario where it makes sense - the primary action is triggered by the dropdown itself.
2. it doesn't work for VM details scenario - `ActionButtonWraper` does not pass tool tip to `DropdownButton` (which by the way seems not to support it anyway). Btw I'm not sure if this bug or feature because in this use case the main button triggers no action.
3. `Console in use` is not translated
4. tool tips for individual console actions seems not to work (all scenarios)

I vote for removing this feature: it is broken anyway (and none complained) .

Reply:
The whole console loading button on the VM List and on VM Details is slated to be updated in the near future anyway.  No need to worry about it here.